catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/validation/impl/validator/EnumerationValidator.java

```diff
@@ -37,6 +37,8 @@
  * Validates an attribute's value(s) against a set of acceptable values.
  */
 public class EnumerationValidator implements AttributeValidator {
+    private final boolean ignoreCase;
```

Comment:
`equals()` and `hashCode()` will have to be updated to handle the case-insensitive option correctly.